Gruntfile.js

```diff
@@ -347,7 +347,7 @@ module.exports = function (grunt) {
             'embed-styles': {
                 [targetName]: {
                     cwd: path.resolve(extensionPath, bundleFolder),
-                    src: '**/*bundle.js',
+                    src: '**/*bundle.js**',
```

Comment:
Maybe a little easier to understand to instead use `['**/*bundle.js', '**/*bundle.js.map']`?